### PR TITLE
add provision for pwa support

### LINKED ISSUE #74: Add provision to have pwa support for Nuxt-js template
Currently, the user can only have the basic template if he chooses `Nuxt-js` from the list.
It should be made such that the user will be prompted whether he/she requires `pwa` support after making the selection as mentioned above. 
See these [guidelines](https://pwa.nuxtjs.org/setup.html)

### PR DESCRIPTION
**Pull request by aquibbaig**
Fixes #74. Please review this and let me know

### CODE REVIEW

File: lib/commands/basic/init.js [hunk 1]
```diff
@@ -68,26 +68,57 @@ let fetchTemplate = (template) => {
...
-      });
+    setTimeout(() => {
+        console.log('\n');
+        //add pwa support
+        inquirer.prompt([{
+          name: 'pwa',
+          type: 'confirm',
+          message: 'Do you want to have pwa support?'
+        }])
+        .then( async(answers) => {
```
Comment (jamesgeorge007) [inline lib/commands/basic/init.js]: It is good to have the installation process after the user fires in `mevn serve` :thinking: 
What intended is to have the user choose all his required features and later proceed with the installation. Hope you got the sense :+1:

**Comment by aquibbaig** [comment]
Uh, I don't know. Is this according to the requirements? I agree that the dependencies should be installed after the serve option. Also pwa support will be asked after Universal/SPA selection

File: lib/commands/serve/setup.js [hunk 2]
```diff
@@ -2,20 +2,22 @@
...
 const shell = require('shelljs');
+const fs = require('fs');
 
 const { appData } = require('../../utils/projectConfig');
 const { configFileExists } = require('../../utils/messages');
 const { serveProject } = require('./launch');
 const { showBanner } = require('../../external/banner');
 
 let projectTemplate;
+let projectName;
```
Comment (aquibbaig) [inline lib/commands/serve/setup.js]: This is not required I guess

File: lib/commands/basic/init.js [hunk 3]
```diff
@@ -66,30 +66,6 @@ let fetchTemplate = (template) => {
...
-      console.log('\n');
-
-      inquirer.prompt([{
-        name: 'mode',
-        type: 'list',
-        message: 'Choose your preferred mode',
-        choices: ['Universal', 'SPA']
-      }])
-      .then((choice) => {
-        if (choice.mode === 'Universal') {
```
Comment (jamesgeorge007) [inline lib/commands/basic/init.js]: It is good to have the `pwa required` prompt here as from the previous commit :+1:

**Comment by jamesgeorge007** [comment]
@aquibbaig Resolve the conflicts

**Comment by aquibbaig** [comment]
@jamesgeorge007 Is the logic alright though?

**Pull request closed without merging**